geodesy/README.md

```diff
@@ -148,9 +148,11 @@ Last, we add some details like beach ball, epicenters, etc to the map.
 
 <img src="demo7.jpg" width="40%">
 
-### Bonus challenge
+### Bonus challenges
 
-Figure out what's inside the final map and complete the rest of the plot. 
+Figure out what's inside the final map and complete the rest of the plot. Focal mechanisms and epicenters could be find here: 7.1 https://earthquake.usgs.gov/earthquakes/eventpage/ci38457511/executive and 6.4 https://earthquake.usgs.gov/earthquakes/eventpage/ci38443183/executive
```

Comment:
could be find here: 7.1 https://earthquake.usgs.gov/earthquakes/eventpage/ci38457511/executive and 6.4

Reply:
Probably should say" can be found here"